internal/policies/manager.go

```diff
@@ -270,6 +270,7 @@ func (m *Manager) DumpPolicies(ctx context.Context, objectName string, withRules
 	// Load target policies
 	policiesTarget, err := NewFromCache(ctx, filepath.Join(m.policiesCacheDir, objectName))
 	if err != nil {
+		log.Infof(ctx, i18n.G("User %q not found on cache. To see policies applied to the machine, use 'adsysctl policy applied --machine'"), objectName)
```

Comment:
Don’t paste commands in code here. Maybe this is not used by a command but by a GUI or other ways, so let’s be agnostic from the CLI.

You can still tell that the "User %q not found in cache" though, I think this is enough.